core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -290,6 +295,8 @@ object TopicCommand extends Logging {
 
     override def describeTopic(opts: TopicCommandOptions): Unit = {
       val topics = getTopics(opts.topic, opts.excludeInternalTopics)
+      ensureTopicExists(topics, opts.topic, !opts.ifExists)
```

Comment:
Same issue here as above.  You can get past this line this way, but you still fail when actually calling `describeTopics`.

Another issue with this code is that we should NOT be listing all partition reassignments.  `Admin#listPartitionReassignments` has a variant that takes a list of partitions.  We should use that so that we're not fetching a lot of information that we don't need.  I think this should be an easy fix

Reply:
I will look into the new API.. so `topics` will be empty and I checked that the describeTopics() will actually send an empty topicList.. Let me look into simplifying this more.

Reply:
On avoiding `getTopics`, same issue.. `AdminClient#describeTopics(..)` cannot handle regexes. I made the changes for using the trimmed down `Admin#listPartitionReassignments`